metals/src/main/scala/scala/meta/internal/metals/Diagnostics.scala

```diff
@@ -191,8 +191,7 @@ final class Diagnostics(
     val edit = TokenEditDistance(
       snapshot,
       current,
-      trees,
-      doNothingWhenUnchanged = false
```

Comment:
No idea why would this be needed, the only thing it would do is to copy the range.